.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thank you for your interest in contributing to the venia-pwa-concept project! Before you start contributing, please take a moment to read through the following guidelines:
+
+* [Code of Conduct]
+* [Support]
+
+To contribute to this repository, start by forking the [official repository] and following the installation instructions in the README file.
+
+## Pull Request checklist
+
+* Code contributions must adhere to the [Coding style guide](#coding-style-guide)
+* PR commits should contain [meaningful commit messages]
+* To help with reviews, your PR should only create/revise a single feature or fix a single issue.
+* If your PR fixes a bug, please provide a step-by-step description of how to reproduce the bug.
+* If your PR addresses an existing issue, please reference that issue in the title or description.
+
+### Coding style guide
+
+**TODO: Write style guide**
```

Comment:
Maybe we should link to [existing coding standards in Devdocs](https://devdocs.magento.com/guides/v2.3/coding-standards/bk-coding-standards.html) here. I know they're not fully up to date, and we need to backlog a more digestible coding style for research org repositories, but right now it's not urgent; we don't have a lot of PHP contribution, and the only thing we care about w/r/t JavaScript is already enforced by the de facto standards of prettier and our lint rules.

Reply:
I'll advocate for dropping `Coding style guide` from here entirely for now. A good portion of our existing recommendations on devdocs are either aimed at those authoring in ES5, or are purely stylistic (which we don't care about because `prettier`).